packages/babel-plugin/src/__tests__/css-builder.test.ts

```diff
@@ -1,6 +1,24 @@
 import { transform } from '../test-utils';
 
 describe('css builder', () => {
+  it('should convert CSS properties to kebab-case', () => {
+    const actual = transform(`
+      import '@compiled/react';
+      <div css={{ backgroundColor: 'red' }} />
```

Comment:
Can you write a test with cssMap + css funcs please 🙏

Reply:
and styled too

(perhaps xcss? not sure if worth it)

Reply:
Sure, I've added the tests